org.eclipse.jdt.ls.core/src/org/eclipse/jdt/ls/core/internal/corext/template/java/PostfixTemplate.java

```diff
@@ -92,10 +108,14 @@ class PostfixPreferences {
 	public static final String NULL_CONTENT = "if (${i:inner_expression(java.lang.Object,array)} == null) {\n" +
 		"\t$${0}\n" +
 	"}";
+	public static final String NOT_CONTENT = "!${i:inner_expression(boolean)}${}";
```

Comment:
I was going to ask you to do `java.lang.Boolean` as well but when I looked at some of the templates on the Eclipse side, they also did just boolean, so I guess it''s fine.